packages/cfd/src/Components/__tests__/cfd-mt5-demo-account-display.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import CFDMT5DemoAccountDisplay from '../cfd-mt5-demo-account-display';
+import { CFD_PLATFORMS } from '@deriv/shared';
+
+//mock cfd-account-card
+jest.mock('../cfd-account-card', () => ({
+    CFDAccountCard: () => <div>CFDAccountCard</div>,
+}));
+
+describe('CFDMT5DemoAccountDisplay', () => {
+    const mock_standpoint = {
+        financial_company: 'financial_company',
+        gaming_company: 'gaming_company',
+        iom: false,
+        malta: false,
+        maltainvest: false,
+        svg: true,
+    };
+
+    const mock_props: React.ComponentProps<typeof CFDMT5DemoAccountDisplay> = {
+        is_eu: false,
+        is_eu_country: false,
+        has_maltainvest_account: false,
+        has_cfd_account_error: false,
+        openAccountNeededModal: () => jest.fn(),
+        standpoint: mock_standpoint,
+        is_loading: false,
+        is_logged_in: true,
+        isSyntheticCardVisible: jest.fn(() => true),
```

Comment:
```suggestion
        isSyntheticCardVisible: jest.fn().mockReturnValue(true),
```

Not. a big deal @hamza-deriv

Reply:
sure 
Done